portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsMultipleSelectionField.testcase

```diff
@@ -43,13 +43,38 @@ definition {
 	}
 
 	@description = "Verify that a Multiple Selection Field can be deleted"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeDeleted {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-132262 CanBeDeleted pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Multiple Selection");
+
```

Comment:
Hi @chaowang-123, can you please add step 5 in the ticket, "Assert the multiple selection field is no longer present" in your test? Thank you!

Reply:
Hi @brittneyq , have added, thank you~